crates/nu-cli/src/config_files.rs

```diff
@@ -3,10 +3,10 @@ use crate::util::eval_source;
 use nu_path::canonicalize_with;
 use nu_protocol::{
     engine::{EngineState, Stack, StateWorkingSet},
```

Comment:
According to ci warning, is `StateWorkingSet` only useful under plugin feature?

Reply:
Oops, missed that one. Fixed.